src/relayFeeCalculator/chain-queries/arbitrum.ts

```diff
@@ -27,9 +27,10 @@ export class ArbitrumQueries implements QueryInterface {
     return estimateTotalGasRequiredByUnsignedTransaction(tx, this.simulatedRelayerAddress, this.provider);
   }
 
-  async getTokenPrice(tokenSymbol: string): Promise<number> {
+  async getTokenPrice(tokenSymbol: string, coingeckoProApiKey?: string): Promise<number> {
```

Comment:
Can this key be passed in the relayerFeeCalculator config?

Reply:
Even better, I can inject into the Query constructor